src/app/core/services/external-cluster.ts

```diff
@@ -234,6 +235,21 @@ export class ExternalClusterService {
     return this._http.get<GKEZone[]>(url, {headers: this._getGKEHeaders()}).pipe(catchError(() => of<[]>()));
   }
 
+  getGKEDiskTypes(projectID: string, clusterID: string): Observable<GCPDiskType[]> {
```

Comment:
```suggestion
  getGKEDiskTypesForMachineDeployment(projectID: string, clusterID: string): Observable<GCPDiskType[]> {
```

- update all methods which are related to `MD` creation
- Inlcuding EKS one to remove `Create` to be consistent in all providers